epan/dissectors/packet-ippusb.c

```diff
@@ -21,70 +21,421 @@
 #include <epan/to_str.h>
 #include <epan/conversation.h>
 #include <epan/wmem/wmem.h>
-#include "packet-http.h"
-#include "packet-usb.h"
+#include <reassemble.h>
+#include <packet-http.h>
+#include <packet-usb.h>
 #include <stdio.h>
 
+#define HTTP 0
+
 void proto_register_ippusb(void);
 void proto_reg_handoff_ippusb(void);
+static int is_http_header(int first_linelen, const guchar *first_line);
+
+/* Reassemble by default */
+static gboolean global_ippusb_reassemble = TRUE;
```

Comment:
>global_ippusb_reassemble [](start = 16, length = 24)

Is this still needed?  I don't see it being referenced.

Reply:
I have now added this to be a part of a check to disable reassembly if the user wants

---
In reply to: [454625097](https://github.com/tjamhar/wireshark/pull/2#discussion_r454625097) [](ancestors = 454625097)